wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/about/AboutScreen.kt

```diff
@@ -17,82 +15,33 @@ import androidx.compose.runtime.Composable
 import com.android.identity_credential.wallet.ui.ScreenWithAppBarAndBackButton
 
 @Composable
 fun AboutScreen(loggerModel: LoggerModel, onNavigate: (String) -> Unit) {
     val localUriHandler = LocalUriHandler.current
     ScreenWithAppBarAndBackButton(
         title = stringResource(R.string.about_screen_title),
-        onBackButtonClick = { onNavigate(WalletDestination.PopBackStack.route) }
+        onBackButtonClick = { onNavigate(WalletDestination.PopBackStack.route) },
+        scrollable = false
     ) {
-        Spacer(modifier = Modifier.weight(0.5f))
         Row(
-            modifier = Modifier.fillMaxWidth(),
+            modifier = Modifier.weight(1.0f),
             horizontalArrangement = Arrangement.Center
         ) {
-            Column() {
-                Text(
-                    modifier = Modifier.padding(8.dp),
-                    style = MaterialTheme.typography.bodyLarge,
-                    text = stringResource(R.string.about_screen_text)
-                )
-                ClickableText(
-                    modifier = Modifier
-                        .padding(8.dp)
-                        .align(Alignment.CenterHorizontally),
-                    style = MaterialTheme.typography.bodyLarge,
-                    text = buildAnnotatedString {
-                        withStyle(
-                            style = SpanStyle(
-                                color = Color.Blue,
-                                fontWeight = FontWeight.Bold
-                            )
-                        ) {
-                            append(stringResource(R.string.about_screen_home_page_text))
-                        }
-                    },
-                    onClick = { _ ->
-                        localUriHandler.openUri(
-                            "https://github.com/openwallet-foundation-labs/identity-credential")
-                    }
-                )
-                ClickableText(
-                    modifier = Modifier
-                        .padding(8.dp)
-                        .align(Alignment.CenterHorizontally),
-                    style = MaterialTheme.typography.bodyLarge,
-                    text = buildAnnotatedString {
-                        withStyle(
-                            style = SpanStyle(
-                                color = Color.Blue,
-                                fontWeight = FontWeight.Bold
-                            )
-                        ) {
-                            append(stringResource(R.string.about_screen_iaca_cert_text))
-                        }
-                    },
-                    onClick = { _ ->
-                        localUriHandler.openUri(
-                            "https://github.com/openwallet-foundation-labs/identity-credential/blob/main/wallet/src/main/res/raw/iaca_certificate.pem")
-                    }
-                )
-            }
+            MarkdownAsset(asset = "about.md",
```

Comment:
My intuition is that this should use `Markdown` (instead of `MarkdownAsset`) and the contents of *about.md* should be a translatable string, no?

Reply:
There is MarkdownAsset and MarkdownText.

I tried to put markdown in strings.xml, but it is a pain - extra layer of escaping and newlines don't really work and indentation is potentially an issue. We can put it as file in res/raw, but that will not work well with images - or whatever we'd want to reference. This is a relevant discussion: https://stackoverflow.com/questions/5427933/localization-of-assets-files - perhaps we should follow one of the suggestion there? I can put a layer of indirection, e.g. I can change it that strings.xml contains a pointer to a localized file in assets (probably would do a separate PR).

Reply:
> I can change it that strings.xml contains a pointer to a localized file in assets (probably would do a separate PR).

Sounds good to me!